### PR TITLE
[ALLUXIO-3246] Fix unit for master client thread pool keep-alive time

### PR DESCRIPTION
**Pull request by madanadit**
https://alluxio.atlassian.net/browse/ALLUXIO-3246

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java [hunk 1]
```diff
@@ -397,7 +398,8 @@ protected void startServingRPCServer() {
         .processor(processor)
         .transportFactory(transportFactory)
         .protocolFactory(ThriftUtils.createThriftProtocolFactory())
-        .stopTimeoutVal((int) Configuration.getMs(PropertyKey.MASTER_THRIFT_SHUTDOWN_TIMEOUT));
+        .stopTimeoutVal((int) TimeUnit.MILLISECONDS
```
Comment (gpang) [inline core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java]: Should we also explicitly set `stopTimeoutUnit` to be `TimeUnit.SECONDS`?

Comment (madanadit) [reply core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java]: @gpang Yeah I thought about that. There is no method in `TThreadPoolServer.Args` to set that :-(

Comment (gpang) [reply core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java]: @madanadit Should we just set it directly, since it is `public`?

Comment (madanadit) [reply core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java]: yes, didn't realize it was public

Comment (madanadit) [reply core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java]: @gpang The thrift method `TThreadPoolServer.createDefaultExecutorService` is strange. It does not respect the time unit (if not set to seconds). So we must set to the unit to seconds 
```
  private static ExecutorService createDefaultExecutorService(Args args) {
    SynchronousQueue<Runnable> executorQueue =
      new SynchronousQueue<Runnable>();
    return new ThreadPoolExecutor(args.minWorkerThreads,
                                  args.maxWorkerThreads,
                                  args.stopTimeoutVal,
                                  TimeUnit.SECONDS,
                                  executorQueue);
  }```

**Pull request merged**